### PR TITLE
Changes to support vcf species 

### PR DESCRIPTION
**Pull request by olaaustine**
[ENSVAR-5449](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5449)
[ENSVAR-5460](https://www.ebi.ac.uk/panda/jira/browse/ENSVAR-5460)

### CODE REVIEW

**Comment by jyothishnt** [comment]
Please could you add reviewers? @olaaustine

File: ensembl/conf/json/macaca_fascicularis_vcf.json [hunk 1]
```diff
@@ -8,10 +8,9 @@
       "assembly": "Macaca_fascicularis_6.0",
       "type": "remote",
       "use_as_source" : 1,
-      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_3/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
-      "chromosomes": [
-        "1","2","3","4","5","6","7","8","9","10","11","12","13","14","15","16","17","18","19","20","X"
-      ]
+      "source_version" : 4,
+      "filename_template" : "https://ftp.ebi.ac.uk/pub/databases/eva/rs_releases/release_4/by_species/macaca_fascicularis/GCA_011100615.1/GCA_011100615.1_current_ids.vcf.gz",
+      "use_seq_region_synonyms": 1
```
Comment (nakib103) [inline ensembl/conf/json/macaca_fascicularis_vcf.json:13]: HI @olaaustine , it seems the json we tested have this value as 0. Can you revert it back -
```suggestion
      "use_seq_region_synonyms": 0
```

File: ensembl/conf/json/taeniopygia_guttata_vcf.json [hunk 2]
```diff
@@ -1,12 +1,16 @@
 {
```
Comment (nakib103) [inline ensembl/conf/json/taeniopygia_guttata_vcf.json:1]: We should keep the gerp config and add new config as an addition.

File: ensembl/conf/ini-files/taeniopygia_guttata.ini [hunk 3]
```diff
@@ -71,8 +71,8 @@ ENSEMBL_SEARCH_IDXS     = [ Domain Family Gene GenomicAlignment Marker Sequence
 
 
 
-VARIATION_PARAM   = rs82660989
-VARIATION_TEXT    = rs82660989
+VARIATION_PARAM   = rs83372914;vf=scaffold_10_arrow_ctg1:11015:A_G:EVA
```
Comment (nakib103) [inline ensembl/conf/ini-files/taeniopygia_guttata.ini:74]: Is it better to have an example on the chromosomes instead of the scaffold.

**Review by nakib103 — APPROVED** [review]
Looks good!~

**Pull request merged**